ansible/configs/sap-integration/post_software.yml

```diff
@@ -0,0 +1,371 @@
+---
+- name: Step 005 - Post software
+  hosts: bastions
+  become: no
+  gather_facts: False
+  environment:
```

Comment:
You're not doing any tasks in this play, there is no reason to set the `KUBECONFIG` var.

Reply:
Done, but this is coming from the previous config so probably needs to be changed there as well